packages/ddg2dnr/lib/requestBlocklist.js

```diff
@@ -0,0 +1,96 @@
+/** @module requestBlocklist */
+
+const { getDomain } = require('tldts');
+
+const { processPlaintextTrackerRule, generateDNRRule } = require('./utils');
+
+const EXPECTED_RULE_KEYS = new Set(['domains', 'reason', 'rule']);
+const PRIORITY = 20000;
```

Comment:
Maybe a comment here so we remember this has to be higher than https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/e48ed182a1c4d2f172a2b3a116e5a74b54e534a3/packages/ddg2dnr/lib/tds.js#L25